File: ConstructionQualityControl.Domain/Dtos/Order/OrderRootReadDto.cs
using System;
using System.Collections.Generic;

namespace ConstructionQualityControl.Domain.Dtos
{
    public class OrderRootReadDto
    {
        public int Id { get; set; }
        public DateTime CreationDate { get; set; }
        public UserReadDto User { get; set; }
        public string Demands { get; set; }
        public decimal PostPaid { get; set; }
        public bool IsCompleted { get; set; }
        public bool IsStarted { get; set; }
        public CityReadDto City { get; set; }
        public List<WorkOfferReadDto> WorkOffers { get; set; }
    }
}

File: UI/angular11/src/app/dtos/order/order-root-read-dto.ts
import { CityReadDto } from "../city/city-read-dto"
import { UserReadDto } from "../user/user-read-dto"
import { WorkOfferReadDto } from "../work-offer/work-offer-read-dto"

export class OrderRootReadDto {
    id: number
    creationDate: Date
    user: UserReadDto
    demands: string
    postPaid: number
    isCompleted: boolean
    isStarted: boolean
    city: CityReadDto
    workOffers: WorkOfferReadDto[]
}